cloud-manager/Makefile

```diff
@@ -0,0 +1,21 @@
+APPVERSION = 0.0.1
+APPBIN  ?= bin/cloud-manager
+GOFLAGS = -mod=vendor -ldflags '-X main.BuildVersion=$(APPVERSION)'
+
+export GO111MODULE=on
+
+all: build
+
+.PHONY: build
+build:
+	@echo "Building cloud-manager..."
+	@CGO_ENABLED=0 go build $(GOFLAGS) -o $(APPBIN) github.com/mattermost/mattermost-cloud-monitoring/cloud-manager/cmd/cloud-manager
```

Comment:
Seems like this needs to be run from a specific directory to work correctly? Maybe we can make it use $GOPATH instead?